splunk-otel-android/src/main/java/com/splunk/rum/SplunkRumBuilder.java

```diff
@@ -113,8 +113,7 @@ public SplunkRumBuilder setRumAccessToken(String rumAuthToken) {
     /**
      * Enables debugging information to be emitted from the RUM library.
      *
-     * <p>This feature is disabled by default. You can enable it by calling this configuration
-     * method with a {@code true} value.
+     * <p>This feature is disabled by default. You can enable it by calling configuration method.
```

Comment:
Alternately (for parity with the other javadocs):

```suggestion
     * <p>This feature is disabled by default. You can enable it by calling this method.
```